api/src/main/java/jakarta/enterprise/util/Shutdown.java

```diff
@@ -0,0 +1,26 @@
+package jakarta.enterprise.util;
+
+/**
+ * <p>
+ * A CDI event with payload of type {@link Shutdown} and qualifier {@link jakarta.enterprise.inject.Any} is
+ * <i>synchronously</i> fired by CDI container during application shutdown.
+ * Applications must never manually fire any events with {@link jakarta.enterprise.util.Shutdown} as payload.
+ * </p>
+ *
+ * <p>
+ * Implementations have to fire this event during CDI container shutdown, but not later than
+ * {@code @BeforeDestroyed(ApplicationScoped.class)}.
```

Comment:
```suggestion
 * {@code @BeforeDestroyed(ApplicationScoped.class) Object}.
```

Reply:
Or perhaps better:

```suggestion
 * the event with qualifier {@code @BeforeDestroyed(ApplicationScoped.class)}.
```

Similarly to other places.